spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -383,16 +385,42 @@ NOTE: App Starter actuator endpoints are secured by default.  You can disable se
 |http://bit.ly/Darwin-SR2-stream-applications-rabbit-docker
 |http://bit.ly/Darwin-BUILD-SNAPSHOT-stream-applications-rabbit-docker
 
-|Kafka 0.11 and above + Maven
+|Apache Kafka + Maven
 |http://bit.ly/Darwin-SR2-stream-applications-kafka-maven
 |http://bit.ly/Darwin-BUILD-SNAPSHOT-stream-applications-kafka-maven
 
-|Kafka 0.11 and above + Docker
+|Apache Kafka + Docker
 |http://bit.ly/Darwin-SR2-stream-applications-kafka-docker
 |http://bit.ly/Darwin-BUILD-SNAPSHOT-stream-applications-kafka-docker
 |======================
 
-The following table lists the available Task Application Starters:
+The following table include the bit.ly links to the available Stream Application Starters based on Spring Cloud Stream 2.1.x
+and Spring Boot 2.1.x:
+
+NOTE: App Starter actuator endpoints are secured by default.  You can disable security by deploying streams with the property `app.*.spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration`.  On Kubernetes refer to the section <<getting-started-kubernetes-probes, Liveness and readiness probes>> to configure security for actuator endpoints.
+
+[width="100%",frame="topbot",options="header"]
+|======================
+|Artifact Type |Stable Release |SNAPSHOT Release
+
+|RabbitMQ + Maven
+|http://bit.ly/Einstein-M2-stream-applications-rabbit-maven
+|http://bit.ly/Einstein-BUILD-SNAPSHOT-stream-applications-rabbit-maven
+
+|RabbitMQ + Docker
+|http://bit.ly/Einstein-M2-stream-applications-rabbit-docker
+|http://bit.ly/Einstein-BUILD-SNAPSHOT-stream-applications-rabbit-docker
+
+|Apache Kafka + Maven
+|http://bit.ly/Einstein-M2-stream-applications-kafka-maven
+|http://bit.ly/Einstein-BUILD-SNAPSHOT-stream-applications-kafka-maven
+
+|Apache Kafka + Docker
+|http://bit.ly/Einstein-M2-stream-applications-kafka-docker
+|http://bit.ly/Einstein-BUILD-SNAPSHOT-stream-applications-kafka-docker
+|======================
+
+The following table include the available Task Application Starters based on Spring Cloud Task 1.2.x and Spring Boot 1.5.x:
```

Comment:
Putting it in a section `Spring Cloud Task App Starters` would help it stand out.